external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -165,12 +170,8 @@ def fit_statistics(self, X: xr.Dataset):
         """
         if self.train_keras_model is not None:
             raise RuntimeError("cannot build, model is already built!")
-        temporary_sample_dim = self.sample_dim_name + "_tmp"
-        if self.sample_dim_name in X.dims:
-            X_tmp = X.rename({self.sample_dim_name: temporary_sample_dim})
-        else:
-            X_tmp = X
-        dataset_2d = stack_non_vertical(X_tmp, self.sample_dim_name)
+        X_tmp = _rename_sample_dim(X, self.sample_dim_name)
+        dataset_2d = stack_non_vertical(X_tmp)
```

Comment:
Why was this changed?

Reply:
The same behavior was kept but moved to a function since I added some code that also used this. But that's now unnecessary after revisions, so I'm reverting it back to how it was before.